lib/knuckle_cluster.rb

```diff
@@ -101,7 +101,7 @@ def select_agent(auto: false)
 
       output_agents
 
-      puts "\nConnect to which agent?"
+      puts "\nConnect to which agent? (INDEX)"
```

Comment:
How about something like the following?

```
Enter the INDEX number and press ENTER to connect:
```

My intention is to make the instruction more explicit and also get rid of the unnecessarily mentioning agents or containers.